ckanext/cloudstorage/cli.py

```diff
@@ -164,6 +179,96 @@ def _fix_cors(args):
         )
 
 
+def _get_unlinked_uploads(return_objects = False):
+    cs = CloudStorage()
+
+    resource_urls = []
+    resource_ids_and_filenames = model.Session.query(
+                                    model.Resource.id,
+                                    model.Resource.url) \
+                                 .filter(model.Resource.url_type == 'upload') \
+                                 .all()
+
+    for id, filename in resource_ids_and_filenames:
+        resource_urls.append(os.path.join(
+                                'resources',
+                                id,
+                                munge_filename(filename)))
+
+    uploads = cs.container.list_objects()
+
+    uploads_missing_resources = []
+    for upload in uploads:
+        if upload.name not in resource_urls:
+            uploads_missing_resources.append(
+                upload if return_objects else upload.name)
+
+    return uploads_missing_resources
+
+
+def _list_unlinked_uploads():
+    uploads_missing_resources = _get_unlinked_uploads()
+
+    if len(uploads_missing_resources):
```

Comment:
```suggestion
    if uploads_missing_resources:
```
empty containers are False and non-empty lists are True in python